docs/styles.md

```diff
@@ -0,0 +1,34 @@
+# Styling content for RAISE
+
+In order to maintain consistency and reliability across all RAISE content, the styling associated with RAISE content is consolidated into a series of css classes located in `src/styles`. 
+
+RAISE Style classes are either written for specific elements, or can be applied to a variety of objects. Those differences are listed below.
+
+## Style Classes 
+
+### Indent (os-raise-indent)
+Add a padding to the left side of an element. If the element already includes a left padding (like an ordered list) the indent will get set to exactly 32px
+**Example**
```

Comment:
I think you need an empty line before this (it currently shows up on the same line as the preceding sentence).

That being said, did you mean for this to be a parent of `Availability` and `Usage`, or a peer? If the former, maybe it should be `#### Example` :man_shrugging:

Reply:
Ah, I was going to ask @rnathuji  - my thought was that under Example we could add an image of what the final styling product looks like, but I wasn't sure where I should host a screenshot I'd taken and/or where I'd attach it in the repo file structure. 

Is there an easier way to embed images into .md files that are displayed on github that you know of? I was thinking I could k12-openstax content upload them 😂

Reply:
Alternatively I can just get rid of the Example section for now.

Reply:
Ah -- screenshots are a nice idea. You can actually just commit them to the repo and reference using a relative path. For example, if you add a file `./docs/static/image1.jpg`, you can then reference it in markdown as `![Alt text string](./static/image1.jpg)`